net-proxy/trojan/trojan-1.8.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="An unidentifiable mechanism that helps you bypass GFW"
+HOMEPAGE="https://github.com/trojan-gfw/trojan"
+SRC_URI="https://github.com/trojan-gfw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~arm ~arm64"
```

Comment:
Please sort.

Reply:
Got it!

Reply:
does commit message looks like this ?
```
net-proxy/trojan: bump version 1.8.1
Signed-off-by: YangMame yangmame@icloud.com
```

Reply:
Is 'YangMame' your real name that you can confirm with government-issued document? I'm sorry if I'm mistaken due to cultural difference but it really looks like a nickname.

Reply:
changed :-)